worlds/yugioh06/Opponents.py

```diff
@@ -0,0 +1,223 @@
+from typing import NamedTuple, Callable, List, Optional, Tuple
+
+from BaseClasses import CollectionState, MultiWorld
```

Comment:
```suggestion
from typing import NamedTuple, List, Optional, Tuple

from BaseClasses import MultiWorld
```
Remove unused imports